licenses/templates/legalcode_40_page.html

```diff
@@ -31,12 +31,29 @@
       {% include 'includes/licenses_header.html' %}
       {% include 'includes/about_cc_and_license.html' %}
       {% include 'includes/use_of_licenses.html' %}
-      {% include 'includes/legalcode_40_license.html' %}  {# <<< THE ACTUAL LICENSE TEXT #}
+    <!-- Here comes the license text -->
+      {% if license.version == "4.0" %}
+        {% include 'includes/legalcode_40_license.html' %}  {# <<< THE ACTUAL LICENSE TEXT #}
```

Comment:
I think we should update these comments to say `{# THE ACTUAL <version> LICENSE TEXT #}`

Reply:
Can do.